libs/ui-watt/src/lib/components/button/+storybook/storybook-button-overview.component.scss

```diff
@@ -14,7 +14,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@use '../watt-button-variables' as buttonVariables;
+$watt-button-min-width: 6.25rem;
+$watt-button-size-normal-height: 2.75rem;
+$watt-button-size-large-height: 3.5rem;
+
+$watt-button-primary-focus-outline: 2px solid var(--watt-color-secondary);
+$watt-button-primary-focus-background: var(--watt-color-primary-dark);
+$watt-button-primary-hover-background: var(--watt-color-primary-dark);
+
+$watt-button-secondary-focus-state: 1px solid var(--watt-color-primary);
+$watt-button-secondary-focus-background: var(--watt-color-primary-light);
+$watt-button-secondary-hover-background: var(--watt-color-primary-light);
+
+$watt-button-text-focus-state: 2px solid var(--watt-color-primary);
+$watt-button-text-hover-color: var(--watt-color-primary-dark);
```

Comment:
issue: the problem with inlining these variables here is that the style between the "real" button and the style of the button in Storybook can get out of sync. We have to remember to adjust these variables in case the style of the "real" button changes. Can we consolidate that by keeping (and updating) the styles in only one place?

Reply:
Made som mixins for this, but i dont like it 😅